cfn-resources/project-ip-access-list/cmd/resource/resource.go

```diff
@@ -48,19 +47,28 @@ func validateModel(fields []string, model *Model) *handler.ProgressEvent {
 // Create handles the Create event from the Cloudformation service.
 func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler.ProgressEvent, error) {
 	setup()
-	_, _ = logger.Debugf("currentModel: %+v, prevModel: %+v", currentModel, prevModel)
 
 	if errEvent := validateModel(CreateRequiredFields, currentModel); errEvent != nil {
 		return *errEvent, nil
 	}
 
-	_, _ = logger.Debugf("currentModel: %+v, prevModel: %+v", currentModel, prevModel)
+	if currentModel.Profile == nil {
+		currentModel.Profile = pointy.String(util.DefaultProfile)
+	}
```

Comment:
From another PR, I think we're using a different pattern for checking here. Ideally, lets standardise. See https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/326/files#r1113257393 for the example.